spec/controllers/replies_controller_spec.rb

```diff
@@ -111,18 +113,6 @@
       expect(flash[:error][:message]).to eq("Your post could not be saved because of the following problems:")
     end
 
-    it "requires valid params with an unread_warned param if not read" do
-      user = create(:user)
-      login_as(user)
-      character = create(:character)
-      reply_post = create(:post)
-
-      expect(character.user_id).not_to eq(user.id)
-      post :create, reply: {character_id: character.id, post_id: reply_post.id}, unread_warned: true
-      expect(response).to redirect_to(post_url(reply_post))
-      expect(flash[:error][:message]).to eq("Your post could not be saved because of the following problems:")
-    end
-
```

Comment:
Due to how `last_seen` only takes effect when there's an unread *reply* in the post, this test didn't break. It's no longer applicable, though, so I've removed it.